hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -18,6 +18,13 @@
 {
   name:               "flash_ctrl",
   one_line_desc:      "Interfaces and manages integrated non-volatile flash memory; supports scrambling, integrity, and secure wipe",
+  one_paragraph_desc: '''
+  The OpenTitan flash controller interfaces the integrated, non-volatile flash memory with software and other hardware components in the system, such as life cycle controller, key manager, and OTP controller.
+  It consists of the open source flash controller as well as of a closed source, proprietary vendor flash wrapper and flash module.
```

Comment:
```suggestion
  It consists of the open source flash controller that interfaces with a third party flash module.
```
Does this make sense? In practical terms the flash module will be proprietary but I don't think that's a hard limitation - it's just something not provided by OT.

Reply:
SGTM, thanks @mundaym

Reply:
This sounds good.